### PR TITLE
Updates to advisory council pages

### PR DESCRIPTION
**Pull request by SBBCornell**
These are recent updates by Stephanie to advisory council pages that should go live asap, plus a few other miscellaneous changes.

### CODE REVIEW

File: source/about/people/institutions_advisory_council.md [hunk 1]
```diff
@@ -27,13 +27,22 @@ Maureen P. Walsh is the Scholarly Sharing Strategist and an Associate Professor
...
+
+
 **Robert Harington (Member, 2023-2026),** representing the Society Group
 
 Robert is Associate Executive Director, Publishing at the American Mathematical Society (AMS).
 Robert is a member of the AMS leadership team, and has the overall responsibility for publishing at the AMS, including books, journals and electronic products. Robert is a "Chef" for the Society for Scholarly Publishing (SSP) blog, The Scholarly Kitchen. Robert serves on the Board of Directors of CHOR Inc., (CHORUS). Robert has worked in non-profit and commercial settings, internationally, with particular experience across the United States, Europe, and Asia (focus on China and Japan). Robert holds a doctorate in biochemistry from the University of Oxford, and a first-class honors degree in chemistry from the University of London.
 
 **Martin Kohler (Member, 2023-2026),** representing Consortium arXiv-DH and HGF, Germany
 
+Martin heads the DESY and European XFEL libraries including DESY’s institutional repository, open access and publishing house. He holds a doctorate in physics from the university of Hamburg.
```
Comment (alisonhofer) [inline source/about/people/institutions_advisory_council.md:40]: DESY and XFEL should be both spelled out once. 
Deutsches Elektronen-Synchrotron
X-Ray Free-Electron Laser

**Review by alisonhofer — COMMENTED** [review]
Online 40 I think DESY and XFEL should be spelled out: 

Deutsches Elektronen-Synchrotron

X-Ray Free-Electron Laser

File: source/about/people/institutions_advisory_council.md (hunk 1 above)
Comment (SBBCornell) [reply source/about/people/institutions_advisory_council.md]: edited

**Comment by SBBCornell** [comment]
> Online 40 I think DESY and XFEL should be spelled out:
> 
> Deutsches Elektronen-Synchrotron
> 
> X-Ray Free-Electron Laser

edited

**Pull request merged**